src/dataMap.js

```diff
@@ -594,7 +594,7 @@ class DataMap {
       for (let i = 0, ilen = sliced.length; i < ilen; i++) {
         out = out[sliced[i]];
```

Comment:
So, maybe the better thing is the first check `out` and if it's not `falsy` use it. With this improvement, we don't need the second check, so inner `if` is useless.
```suggestion
        out = out && out[sliced[i]];
```